src/components/Table/Table.tsx

```diff
@@ -2,10 +2,9 @@ import { Table as MantineTable, TextInput } from "@mantine/core";
 import { type ChangeEvent, type ReactNode, useMemo, useState } from "react";
 import { IconSortAscending, IconSortDescending } from "@tabler/icons-react";
 import { omit } from "lodash/fp";
-import { isEmpty } from "lodash";
-import { getNextSortValue } from "../../utils/utils.ts";
+import { filterData, getNextSortValue, sortData } from "../../utils/utils.ts";
 
-interface Column {
+export interface Column {
```

Comment:
this interface cal also accept the generic, and then you won't need to use `any` in the `cellRenderer`